### ISSUE TITLE
Add special rendering for shop=charity

### ISSUE THREAD

**Issue by kocio-pl** [gravitystorm/openstreetmap-carto#1869]
shop=charity has more than 2,5k uses and a wiki page. We already show it as a general dot. 

My draft:
1) ![charity-14-17](https://cloud.githubusercontent.com/assets/5439713/10079048/34130fcc-62e8-11e5-904c-bd812ef040c8.png)

**Comment by math1985** [comment]
An horizontal open hand is also often used as symbol for charity.

**Comment by kocio-pl** [comment]
I'll try it, thanks for the idea! In the meantime different take:
2) ![charity-14-19](https://cloud.githubusercontent.com/assets/5439713/10096296/aec5e978-6370-11e5-9f7a-e827a36324d1.png)

**Comment by math1985** [comment]
Unfortunately none of the presented examples are very clear to me - what do others think?

**Comment by kocio-pl** [comment]
I still plan to test your idea, however had no time lately.

**Comment by pnorman** [comment]
> Unfortunately none of the presented examples are very clear to me

Likewise. In order of preference, they would be 1, 2, and 3, but I don't like any of them.

**Comment by kocio-pl** [comment]
I guess this is not something worth exploring at this moment.

**Comment by lakedistrictOSM** [comment]
Another way of looking at charity shops is that they are essentially a reuse/recycle point for clothes, books etc - what would the department_store icon look like if the **&** was replaced with the recycling icon (or a simplified version)? I'm having problem with Inkscape at the moment so can't test this myself.

If this works it could be used for shop=second_hand too.

(charity is now 5k and second_hand is 6k)

**Comment by kocio-pl** [comment]
I can't imagine it, so some real design would be needed to test it. Just the current recycle bin icon in violet would be probably not distinct enough.

**Comment by lakedistrictOSM** [comment]
Not just the recycle bin icon but with a bag or basket. This was what I had in mind, but the recycling icon doesn't work well at a smaller size: ![charitydraft1png](https://user-images.githubusercontent.com/20193716/37172000-7f357b52-2307-11e8-853a-17e90bbd8def.png) ![charitydraft2png](https://user-images.githubusercontent.com/20193716/37172239-20522d1e-2308-11e8-85de-0ec4c8816732.png)

I have some other ideas for charity shop icons as well which I might try out when I have some free time.

**Comment by kocio-pl** [comment]
The right one looks quite good to me. Could you post a gist with them?

**Comment by lakedistrictOSM** [comment]
I only saved a PNG version because I didn't like it - but it's basically just the department_store icon with the "&" replaced with the recycling icon, as big as it can be inside the bag shape. Sorry @kocio-pl !

**Comment by kocio-pl** [comment]
@Tomasz-W What do you think about it?

**Comment by Tomasz-W** [comment]
For me, some 'refresh' symbol would fit here better. A 'recycling' symbol is too much associated with ecology topic, so it may be confusing.

http://simpleicon.com/wp-content/uploads/refresh.png

I can try to combine it with bag shape for shop=charity
https://github.com/gravitystorm/openstreetmap-carto/blob/master/symbols/doctors.svg

and t-shirt shape for shop=second_hand
https://github.com/gravitystorm/openstreetmap-carto/blob/master/symbols/shop/clothes.svg

**Comment by kocio-pl** [comment]
Great, nice idea! So I reopen the issue.

**Comment by Tomasz-W** [comment]
* shop=charity:
![shop charity](https://user-images.githubusercontent.com/25656654/37251916-e73a228a-2518-11e8-8abd-745fe9603732.png)

* shop=second_hand
![shop second_hand](https://user-images.githubusercontent.com/25656654/37251949-80dde98a-2519-11e8-8cc6-92a774606cc1.png)

For shop=second_hand we could consider to use a shop=clothes icon, because they are quice simillar objects.

**Comment by lakedistrictOSM** [comment]
Both good designs, shop=charity is clearer for me. 

shop=second_hand don't necessarily sell clothes, four second_hand shops near me don't sell any clothes but mainly books, [bric-a-brac](https://en.wikipedia.org/wiki/Bric-%C3%A0-brac) and antiques. (This is in the UK, I imagine what a second_hand shop sells varies worldwide)

I'd consider shop=charity and shop=second_hand to be more similar ad therefore the same icon could be used for both.

**Comment by kocio-pl** [comment]
I agree with @lakedistrictOSM - the first icon is good, the second one looks less clear and maybe we could use it for `shop=clothes+second_hand=yes/only`, but for both shop=charity/second_hand the first one should be used.

**Comment by lakedistrictOSM** [comment]
@Tomasz-W Could you also make a version with a square handle like variety_store, just for comparison?
![variety_store](https://wiki.openstreetmap.org/w/images/2/24/Variety_store-14.svg)

**Comment by dieterdreist** [comment]
sent from a phone

> On 11. Mar 2018, at 13:41, lakedistrictOSM <notifications@github.com> wrote:
> 
> shop=second_hand don't necessarily sell clothes, four second_hand shops near me don't sell any clothes but mainly books, bric-a-brac and antiques. (This is in the UK, I imagine what a second_hand shop sells varies worldwide)
> 


shop=second_hand is somehow the corresponding shop to department store, i.e. it sells more than one typology of goods, otherwise the wiki says you should use second_hand as a property 


> I'd consider shop=charity and shop=second_hand to be more similar ad therefore the same icon could be used for both.
> 


“charity” is IMHO not a good category in general, as it remains unclear what they sell, it rather seems to refer to the operator type of the business.

**Comment by Tomasz-W** [comment]
![shop charity1](https://user-images.githubusercontent.com/25656654/37270766-5cf9a0ce-25d0-11e8-9e5c-34d9b837949d.png) (round handle)
![shop charity](https://user-images.githubusercontent.com/25656654/37270481-442f4ebe-25cf-11e8-9db3-9e53db9bb1e8.png) (square handle)

Both versions in this link:   https://gist.github.com/Tomasz-W/23392e6d302279f419dcad435eb7de02

**Comment by Hjart** [comment]
JOSM has an icon for [charity shop](https://wiki.openstreetmap.org/wiki/Tag:shop%3Dcharity):
![josmcharity](https://user-images.githubusercontent.com/9010705/37283547-a913455a-25f8-11e8-9d59-d1a052439a33.png)
It's at https://josm.openstreetmap.de/browser/josm/trunk/images/presets/shop/charity.svg

In my experience shop=charity represents a very distinct category of shops. I frequently visit several myself and in e.g. Denmark they are extremely common.
Charity shops are different from second hand shops in general in that they are run by charities, usually entirely staffed by volunteers and only sells whatever is donated to them by the public

**Comment by kocio-pl** [comment]
Of course they are distinct (that's why we have different tags), but so is meadow, grass and grassland - yet we render them the same, because they are similar. Both shops usually sell used things and for depicting this fact the icon is relevant.

**Comment by Hjart** [comment]
I'm ok with using the same icon for both

My rant was in response to

@dieterdreist 
> “charity” is IMHO not a good category in general, as it remains unclear what they sell, it rather seems to refer to the operator type of the business.

Being quite familiar with both types of shops I disagree with this.

**Comment by dieterdreist** [comment]
sent from a phone

> On 12. Mar 2018, at 16:11, Michael Andersen <notifications@github.com> wrote:
> 
> Being quite familiar with both types of shops I disagree with this.
> 


your text seems to confirm my statement though.

**Comment by Hjart** [comment]
@dieterdreist Does the part of my text saying 

> Also if you have visited a few of these, you'll have a pretty good idea of the range of goods sold in all of them.

confirm your statement?

To me shop=charity is quite a useful category, as it tells me quite accurately what to expect when stepping into one of them.

Consider shop=supermarket. You'll have to be familiar with 'supermarkets' to know what they are selling too, since the description doesn't directly indicate it (like i.e. shop=car does). They don't actually sell supermarkets. There's also quite a wide variety between individual supermarket chains (much more than between charity shops). Some will satisfy fairly basic everyday needs only, while larger ones sells i.e. clothes and furniture too. So, in order to know what to expect, you'll have to be familiar not only with the concept of 'supermarket', but also with individual chains.

**Comment by lakedistrictOSM** [comment]
@Tomasz-W Good idea 👍

**Comment by jragusa** [comment]
For consistency with `shop=variety_store`and `shop=travel_agency`, I would suggest square handle

**Comment by Tomasz-W** [comment]
Gist links:
* shop=charity
![clean svg shop charity kwadrat2](https://user-images.githubusercontent.com/25656654/42150048-c886451c-7dd8-11e8-9a05-dded2384614e.png)
https://gist.github.com/Tomasz-W/72b04561b7aa48b03b790b7720d5fc1b
* shop=second_hand
![clean svg shop second_hand kwadrat2](https://user-images.githubusercontent.com/25656654/42150064-d3739e5c-7dd8-11e8-852b-6477edcb6abd.png)
https://gist.github.com/Tomasz-W/8bf786c909e21ac23d7f39f857ae45bb

**Comment by Hjart** [comment]
Different shapes makes them easier to tell apart :-)

**Comment by meased** [comment]
> For consistency with shop=variety_store and shop=travel_agency, I would suggest square handle

Don't forget about `shop=department_store`, which already has the round handles. More of a meta issue, but I think all "shopping bag" icons should follow the same theme (the same bag with different symbols on it) instead of having four bags that are all slightly different.

Personally, I like the look of the round handles.

**Comment by jragusa** [comment]
@meased Indeed, I missed this icon. I agree with you. We should have the same pattern for all of these icons

**Comment by Tomasz-W** [comment]
@jragusa Departament store icon on "square handle" template:

![shop departament_store](https://user-images.githubusercontent.com/25656654/42221968-2c9e94c2-7ed4-11e8-97ac-d29ffa9322a5.png)

Gist link: https://gist.github.com/Tomasz-W/85550c86782cc47def1d61fd033ca75b
